budgetportal/provincial_infrastructure_projects.py

```diff
@@ -0,0 +1,120 @@
+from tablib import Dataset
+
+
+NORMAL_HEADERS = [
+    "Project ID",
+    "Project No",
+    "Project Name",
+    "Province",
+    "Department",
+    "Local Municipality",
+    "District Municipality",
+    "Latitude",
+    "Longitude",
+    "Project Status",
+    "Project Start Date",
+    "Estimated Construction Start Date",
+    "Estimated Project Completion Date",
+    "Contracted Construction End Date",
+    "Estimated Construction End Date",
+    "Professional Fees",
+    "Construction Costs",
+    "Variation Orders",
+    "Total Project Cost",
+    "Project Expenditure from Previous Financial Years (Professional Fees)",
+    "Project Expenditure from Previous Financial Years (Construction Costs)",
+    "Project Expenditure from Previous Financial Years (TOTAL)",
+    "Main Budget Appropriation (Professional Fees)",
+    "Adjustment Budget Appropriation (Professional Fees)",
+    "Main Budget Appropriation (Construction Costs)",
+    "Adjustment Budget Appropriation (Construction Costs)",
+    "Main Budget Appropriation (TOTAL)",
+    "Adjustment Budget Appropriation (TOTAL)",
+    "Main Budget Appropriation (Professional Fees)",
+    "Adjustment Budget Appropriation (Professional Fees)",
+    "Main Budget Appropriation (Construction Costs)",
+    "Adjustment Budget Appropriation (Construction Costs)",
+    "Main Budget Appropriation (TOTAL)",
+    "Adjustment Budget Appropriation (TOTAL)",
```

Comment:
looks like duplication from row 27-32 which we've had in earlier data

Reply:
yes, that columns were duplicate in the example spreadsheet.

Reply:
solved

https://github.com/vulekamali/datamanager/pull/189/commits/bdbc08c77a50811eac41d1b4d18493fc71a1c3af